src/pou.json

```diff
@@ -0,0 +1,34 @@
+{
+  "DSP": {
+    "Server": "api.0827-test-server.dasch.swiss",
+    "Ontology": "pou",
+    "ShortCode": "0827",
+    "ShortName": "pou",
+    "User": {
+      "Email": "pou0827import@example.com",
+      "Pwd": "test"
+    }
+  },
+  "Header": {},
+  "Footer": {},
+  "Page": [
+    {
+      "URI": "???",
```

Comment:
If you add an URI here, you can test the output on the subpage. Say you add "URI": "test" you can check on localhost:port/#/test

Reply:
Will happen later

Reply:
Will come later